### ISSUE TITLE
Release usethis 1.5.0

### ISSUE THREAD

**Issue by jennybc** [r-lib/usethis#672]
Prepare for release: 
 
* [ ] `devtools::check()` 
* [ ] `devtools::check_win_devel()` 
* [ ] `rhub::check_for_cran()` 
* [ ] `revdepcheck::revdep_check(num_workers = 4)` 
* [ ] [Polish NEWS](https://style.tidyverse.org/news.html#news-release) 
* [ ] Polish pkgdown reference index 
* [ ] Draft blog post 
 
Submit to CRAN: 
 
* [ ] `usethis::use_version('minor')` 
* [ ] Update `cran-comments.md` 
* [ ] `devtools::submit_cran()` 
* [ ] Approve email 
 
Wait for CRAN... 
 
* [ ] Accepted :tada: 
* [ ] `usethis::use_github_release()` 
* [ ] `usethis::use_dev_version()` 
* [ ] Finish blog post 
* [ ] Tweet 
* [ ] Add link to blog post in pkgdown news menu

**Comment by coatless** [comment]
Is there an ETA on when this will go live? Is it safe to say Mid-April? Trying to update the _R_ package lecture material.

**Comment by jennybc** [comment]
Yeah, I definitely want to have it released by mid-April! No promises, but it sounds like our hopes coincide.

**Comment by jennybc** [comment]
@coatless @ijlyttle I know you both have a time-sensitive interest in this. usethis v1.5.0 is now on CRAN.

**Comment by ijlyttle** [comment]
Thanks so much, this is awesome! In my case, sending out prep materials today 🎉
